freyja/data/plot_config.yml

```diff
@@ -0,0 +1,32 @@
+# Lineages: (Lineages key makes changes to the "Lineages" and "Load Scaled Lineages" plot. The Lineages is optional.)
+#   key1:
+#    name: "Lineage 1" (The name key changes the name on the legend on the graph.)
+#    members:
+#     - member1 (The lineages under the members key will be aggreagated to gether and represented as a single plot on the graph.)
+#     - member2* (Added a '*' to the end of the lineage name indicates that the sub-lineages of that lineage have to be included too.)
+#    color: "red" (The color key changes the color of the plot on the graph.)
+# VOC: (VOC key makes changes to the "VOC plot" and "Load Scaled Summary" plot. The VOC key is optional.)
+#   key1:
+#    name: "VOC 1" (The name key changes the name on the legend on the graph.)
+#    color: "red" (The color key changes the color of the plot on the graph.)
+# Please refer to https://github.com/cov-lineages/lineages-website/blob/master/data/lineages.yml
+# to see the relationship between lineages.
+# Note: if 2 keys in the 'Lineages' key have overlapping members, the one with the highest priority will be used.
+# The priority is determined by the order of the keys in the 'Lineages' key.
+Lineages:
+  AY.48:
+    name: "A"
```

Comment:
While I know the user is free to do as they please here,  this is a bit confusing. AY.48 is a delta lineage, BA.2.12 is omicron and A is one of the initial two clades of the epidemic. Probably better to keeps these all consistent (and probably BA.2.12 focused)

Reply:
Fixed.
I change it to:
```yaml
Lineages:
  grp_1:
    name: "grp_1"
    members:
      - "BA.2.12*"
    color: "orange"
  grp_2:
    name: "grp_2"
    members:
      - "BA.2*"
    color: "green"
```

Reply:
Looks good! Thanks!